llvm/utils/lit/tests/shtest-cat.py

```diff
@@ -0,0 +1,31 @@
+## Test the cat command
+
+# RUN: not %{lit} -a -v %{inputs}/shtest-cat \
+# RUN: | FileCheck -match-full-lines %s --dump-input-context=1000
```

Comment:
Why is `--dump-input-context=1000` needed?

Reply:
Oh, that was just to get more context as I was debugging. I can remove that in my next patch